openreview/venue/venue.py

```diff
@@ -1073,50 +1073,49 @@ def ithenticate_create_and_upload_submission(self):
             self.iThenticatePlagiarismCheckApiBaseUrl,
         )
 
+        edges = self.client.get_grouped_edges(invitation=self.get_iThenticate_plagiarism_check_invitation_id(), groupby='head')
+        edges_dict = {edge["id"]["head"]: edge["value"] for edge in edges}
```

Comment:
I think the keyword here is `values` instead of `value`

Reply:
Done